meteor/sofie-server/model/entities/rundown.ts

```diff
@@ -139,7 +139,7 @@ export class Rundown extends BasicRundown {
 	}
 
 	public deactivate(): void {
-		this.assertActive(this.deactivate.name)
```

Comment:
Why?

Reply:
No idea, i did not touch that :-D Im changing it back!